lib/services.js

```diff
@@ -168,6 +169,14 @@ const services = {
         // will reassign some of the above values with the ones
         // from the intiation of the multipart upload
         // (for instance, storage class)
+        const validateUserMdRes = validateUserMetadata(metaHeaders);
```

Comment:
Will this method eventually handle multiple errors? If not, it might be clearer to have it return a boolean.